subscription_manager.go

```diff
@@ -217,7 +224,9 @@ func (sm *SubscriptionManager) Put(ctx context.Context, name string, opts ...Sub
 		return nil, err
 	}
 
-	reqBytes = xmlDoc(reqBytes)
+	str := string(reqBytes)
+	str = strings.Replace(str, `xmlns:XMLSchema-instance="`+schemaInstance+`" XMLSchema-instance:type`, `xmlns:i="`+schemaInstance+`" i:type`, -1)
```

Comment:
Why is this necessary?

Reply:
Service Bus will return a `"400, Details: The specified resource description is invalid"` if `DefaultRuleDescription` has a xmlns named `XMLSchema-instance` for some reason.

Same workaround is applied in the `putRule` function with this comment attached:

[TODO: fix the unmarshal / marshal of xml with this attribute or ask the service to fix it. This is sad, but works.](https://github.com/Azure/azure-service-bus-go/blob/d452ef94455eae1e3822ef47413cfcdbfa512e15/subscription_manager.go#L403)